int16 batteryID
int16 batteryhealth 
time stamp
